source/Plugins/ExpressionParser/Swift/SwiftASTManipulator.h

```diff
@@ -63,7 +63,7 @@ class SwiftASTManipulatorBase {
 
     VariableInfo(CompilerType &type, swift::Identifier name,
                  VariableMetadataSP metadata, 
-                 swift::VarDecl::Specifier specifier = swift::VarDecl::Specifier::Let,
+                 swift::VarDecl::Specifier specifier = swift::VarDecl::Specifier::Var,
```

Comment:
Ah, we have the convention that if you are calling a function with a bunch of arguments, instead of doing:

`ComplexFunction(false, false, true, false);
`
we do

```
bool does_thing_a = true;
bool does_thing_b = false;
...
ComplexFunction(does_thing_a, does_thing_b, ...)

```

To make the code easier to read.  Somebody was being lazy and inlining the variable definition.  That was an awkward way to write that in the first place, but also, since you are passing this enum and not a "true" or "false" it documents itself so the variable isn't necessary.

Can you remove that and then this is fine.